### ISSUE TITLE
An error occurred: BastionSecurityGroup - CIDR block ##.##.##.###   /32 is malformed

### ISSUE THREAD

**Issue by cbarlow1993** [smoketurner/serverless-vpc-plugin#390]
Looks like an interesting plugin and I'm trying to give it a go. 

Using version: "serverless-vpc-plugin": "^1.0.0"

My config looks like this but get an error thrown:
```
  An error occurred: BastionSecurityGroup - CIDR block ##.##.##.###
  /32 is malformed (Service: AmazonEC2; Status Code: 400; Error Code: InvalidParameterValue; Request ID: #########-####-####-####-###########; Proxy: null).
```

```
 vpcConfig:
    cidrBlock: '10.0.0.0/16'
 
    # if createNatGateway is a boolean "true", a NAT Gateway and EIP will be provisioned in each zone 
    # if createNatGateway is a number, that number of NAT Gateways will be provisioned 
    # createNatGateway: 2
 
    # When enabled, the DB subnet will only be accessible from the Application subnet 
    # Both the Public and Application subnets will be accessible from 0.0.0.0/0 
    createNetworkAcl: true
 
    # Whether to create the DB subnet 
    createDbSubnet: true
 
    # Whether to enable VPC flow logging to an S3 bucket 
    createFlowLogs: false
 
    # Whether to create a bastion host 
    createBastionHost: true
    bastionHostKeyName: desktop # required if creating a bastion host 
 
    # Whether to create a NAT instance 
    createNatInstance: true
 
    # Whether to create AWS Systems Manager (SSM) Parameters 
    createParameters: true
 
    # Optionally specify AZs (defaults to auto-discover all availabile AZs) 
    zones:
      - eu-central-1a
      - eu-central-1b
 
    # By default, S3 and DynamoDB endpoints will be available within the VPC 
    # see https://docs.aws.amazon.com/vpc/latest/userguide/vpc-endpoints.html 
    # for a list of available service endpoints to provision within the VPC 
    # (varies per region) 
    services:
      - kms
      - secretsmanager
 
    # Optionally specify subnet groups to create. If not provided, subnet groups 
    # for RDS, Redshift, ElasticCache and DAX will be provisioned. 
    subnetGroups:
      - rds
 
    # Whether to export stack outputs so it may be consumed by other stacks 
    exportOutputs: true
```

Any thoughts on what is going on?

**Comment by jplock** [comment]
https://github.com/smoketurner/serverless-vpc-plugin/issues/318 was submitted for the same issue.  When enabling the bastion host, it attempts to look up your public IP address by querying https://checkip.amazonaws.com.  Would there be any reason the plugin can't reach that address?

**Comment by jplock** [comment]
This should be fixed in `v1.0.1` now
